.github/workflows/ci-check.yml

```diff
@@ -1,5 +1,9 @@
 name: CI Check
-on: push
+on: 
+  push:
+  pull_request:
+    types: [opened]
```

Comment:
This will also allow PRs from forks to go through the CI check without additional pushes.